fundamentals/installation.md

```diff
@@ -2,106 +2,125 @@
 
 Welcome to the **napari** installation tutorial!
 
-This tutorial will teach you how to do a clean install of **napari**. It is aimed at people that just want to use napari. For people also interested in contributing to napari please check our [contributing guidelines](https://github.com/napari/napari/blob/master/docs/developers/CONTRIBUTING.md) for more advanced installation procedures. At the end of the tutorial you should have napari successfully installed on your computer and be able to make the napari viewer appear.
+This tutorial will teach you how to do a clean install of **napari**. It is
+aimed at people that just want to use napari. For people also interested in
+contributing to napari please check our [contributing
+guidelines](https://github.com/napari/napari/blob/master/docs/developers/CONTRIBUTING.md)
+for more advanced installation procedures. At the end of the tutorial you should
+have napari successfully installed on your computer and be able to make the
+napari viewer appear.
 
 ## installation
 
-**napari** can be installed on most macOS, Linux, and Windows systems with Python 3.6 or 3.7. There are three different ways to install napari.
+### from pip, with "batteries included"
 
-### install the latest release from pip
- The simplest option is to install the latest release on PyPi by calling
+napari can be installed on most macOS, Linux, and Windows systems with
+Python 3.6, 3.7 and 3.8 using pip:
 
 ```sh
-$ pip install napari
+pip install napari[all]
+```
+
+Note: while not strictly required, it is *highly* recommended to install
```

Comment:
Can we put this note just above `pip install napari[all]` so the steps are in chronological order, even if one of the steps is optional?

Reply:
I always found libraries that made you work with environments much harder to get going with before I realized it was optional. It does make sense to change the order, but we've got the other order on the main readme right now. Maybe we wait on this and then change it in both places if we want